index.bs

```diff
@@ -96,7 +98,7 @@ we have prepared [a list of these questions in Markdown](https://raw.githubuserc
 <h3 id=resources>Additional resources</h3>
 
 This document is only one of the tools you should use to inform your
-consideration of privacy and security issues in your specifications.
+consideration of privacy and security issues in your specs.
```

Comment:
Using "spec" instead of "specification" in most places seems good to me, though the first use of "spec" instead of "specification" should probably be marked up with an `abbr` element like so:

```html
<abbr title=specification>spec</abbr>
```